server/chat-commands/info.ts

```diff
@@ -2110,10 +2112,16 @@ export const commands: ChatCommands = {
 			// Special case for Meowstic-M
 			if (id === 'meowstic') id = 'meowsticm' as ID;
 			if (['ou', 'uu'].includes(formatId) && generation === 'sm' &&
-				room?.language && room.language in supportedLanguages) {
+				room?.settings.language && room.settings.language in supportedLanguages) {
 				// Limited support for translated analysis
 				// Translated analysis do not support automatic redirects from a id to the proper page
-				this.sendReplyBox(Chat.html`<a href="https://www.smogon.com/dex/${generation}/pokemon/${id}/${formatId}/?lang=${supportedLanguages[room.language]}">${generation.toUpperCase()} ${formatName} ${pokemon.name} analysis</a>, brought to you by <a href="https://www.smogon.com">Smogon University</a>`);
+				this.sendReplyBox(
+					Chat.html`
+					<a href="https://www.smogon.com/dex/${generation}/pokemon/${id}/${formatId}/?lang=
+					${supportedLanguages[room.settings.language]}">
+					${generation.toUpperCase()} ${formatName} ${pokemon.name} analysis</a>,
+					brought to you by <a href="https://www.smogon.com">Smogon University</a>`
+				);
```

Comment:
Um, does `Chat.html` filter out newlines? If not, this might not work the way it was intended. Either way, we're generally not fond of multiline strings here at PS.

```suggestion
				this.sendReplyBox(
					Chat.html`<a href="https://www.smogon.com/dex/${generation}/pokemon/${id}/${formatId}/?lang=${supportedLanguages[room.language]}">${generation.toUpperCase()} ${formatName} ${pokemon.name} analysis</a>, brought to you by <a href="https://www.smogon.com">Smogon University</a>`
				);
```